packages/okta-react/src/OktaError.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React from 'react';
+
+const OktaError = ({ error }) => { 
+  if(error.name && error.message) { 
+    return <p>{error.name}: {error.message}</p>;
```

Comment:
why don't we want to use error.toString()? Doesn't it produce the same result?

Reply:
Not in one case, at least. (I was seeing 'OktaApiError' but no text), plus this pseudo-documents how someone would write their own override.